specs/lifespan.rst

```diff
@@ -0,0 +1,55 @@
+=================
+Lifespan Protocol
+=================
+
+**Version**: 2.1 (2018-07-01)
+
+The Lifespan ASGI sub-specification outlines how to communicate
+lifespan events such as startup and cleanup within ASGI. The lifespan
+being refered to is that of main event loop.
```

Comment:
Can we get a clearer definition of what this means in a multi-process environment? E.g. a main coordinator process and worker processes?

Reply:
I've tried making this clearer.